backend/src/main/java/com/bikes/backend/service/BikeService.java

```diff
@@ -27,15 +27,15 @@ public Bike getBikeById(String id) {
 	}
 
 	public Bike addBike(BikeDTO incomingBike) {
-		Bike bikeToAdd = new Bike(idService.generateId(), incomingBike.title());
+		Bike bikeToAdd = new Bike(idService.generateId(), incomingBike.title(), null);
```

Comment:
u can use a default image at this point, instead of using "null". Maybe a logo which says "no image" or something like that

Reply:
👍